agent/doctor/doctor.go

```diff
@@ -54,19 +68,49 @@ func (doc *Doctor) RunHealthchecks() bool {
 	return doc.allRight(allChecksResult)
 }
 
+// GetPublishInstanceStatusRequest will get all healthcheck statuses and generate
+// a sendable PublishInstanceStatusRequest
+func (doc *Doctor) GetPublishInstanceStatusRequest() (*ecstcs.PublishInstanceStatusRequest, error) {
+	metadata := &ecstcs.InstanceStatusMetadata{
+		Cluster:           aws.String(doc.cluster),
+		ContainerInstance: aws.String(doc.containerInstanceArn),
+		RequestId:         aws.String(uuid.NewRandom().String()),
```

Comment:
Should we log this request ID if it will help in tracking the request ?

Reply:
this will result in some verbose logging, even at the debug level.  And we can use the timestamp in a similar way to troubleshoot.